### DISCUSSION TITLE
A rust beginner here!

### DISCUSSION THREAD

**Discussion by arpancodes** [SeaQL/summer-of-code#1] [Q&A] [answered]
Hey! I am a pre-final year Computer Science Student... I have a bit of knowledge of Rust! I was recently selected for Github Externship during which I have been working on Rust based blockchain substrate. and I would like to think I am a fast learner. 

I have also worked with GraphQL before (on the react part of it, so consuming APIs).

I want to know how much experience in Rust and GraphQL is needed to apply for the first problem statement "A GraphQL framework on top of SeaORM".

**Comment by tyt2y3** [comment, marked as answer]
Welcome! We are updating our contributor's guide.
https://github.com/SeaQL/summer-of-code/blob/main/2022/CONTRIBUTING.md

**Comment by billy1624** [comment]
Hey @arpancodes, GSoC contributor application now opens. Please submit your proposal early so we can review and provide feedbacks :)
